dotCMS/src/main/java/com/dotcms/security/apps/ParamDescriptor.java

```diff
@@ -8,15 +8,15 @@
  * Class used to collect properties or secrets set from the front-end
  * This is mostly used to pass values from the front-end into the Resource.
  */
-public class Param extends AbstractProperty<String>{
+public class ParamDescriptor extends AbstractProperty<String>{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)